plugins/recruiting-impl/src/components/candidates/CreateApplication.svelte

```diff
@@ -112,11 +110,20 @@
 
   let canSave: boolean = false
   $: canSave = !!(recruiter && candidate && stateID)
+
+  function getAvaibleVacancies (applicants: Applicant[]): DropdownItem[] {
```

Comment:
typo, and why not to just define `availableVacancies` and update it by `$`? Also I think that vacancies should be empty until we receive applicants

Reply:
Typo fixed. Because we don't have `not in` query and query updater not update callback func

Reply:
@BykhovDenis I mean something like this:
```ts
let availableVacancies = []
$: if (applicantsLoaded) {
  availableVacancies = vacancyItems
    .filter((item) => !applicants
      .map((app) => app.space)
      .includes(item.id as Ref<Space>))
}
```